app/views/admin/editions/_image_fields_case_studies.html.erb

```diff
@@ -0,0 +1,64 @@
+<fieldset class="image_section">
+  <div class="radio">
+    <%= form.label :image_display_option, (form.object.image_display_option), class: 'radio' do %>
```

Comment:
Think this needs to be a [`labelled_radio_button`](https://github.com/alphagov/whitehall/blob/774eb78cd4c75622e786bdb18762850cd0b91840/lib/whitehall/form_builder.rb#L17) instead - although replacing the entire line with `<label>` works as well and is more readable (line 7 needs to be `</label>` in that case).

This fixes the accessibility such that you can click anywhere on the text to select the radio (you don't have to click within the radio circle).

Reply:
Cool, so this works but you've got a redundant label in there at the moment - if you inspect the HTML (Right Click -> View Source) you'll see something like:

```html
<label class="radio" for="bla">
  <label>
    <input type="radio" />
    ...etc
  </label>
</label>
```

The outer label can be removed, i.e. lines 3 and 9 of this file (and the same lines in the subsequent radio sections below).